modules/bitcoind.nix

```diff
@@ -227,10 +227,7 @@ in {
         type = types.listOf types.str;
         default = [];
         example = [ "ecoc5q34tmbq54wl.onion" ];
-        description = ''
-          Add nodes to connect to and attempt to keep the connections open. Will
-          add one addnode= entry to the config per array element.
-        '';
+        description = "Add nodes to connect to and attempt to keep the connections open. Will add one addnode= entry to the config per list element.";
```

Comment:
For long descriptions multi-line is better :smiley:

Reply:
Oops I mistook the line you were commenting on above. Fixed.